src/Host.cpp

```diff
@@ -1824,7 +1824,11 @@ bool Host::uninstallPackage(const QString& packageName, int module)
             return false;
         }
     }
-
+    //module == 2 seems to be only used for reloading/syncing, which doesn't work for mpackages anyway
```

Comment:
Yeah - for reference the values are - I *believe*:

0. Load as a package
1. Load as a module (from the module/package managers)
2. (Re)load as a module being synced (updated in another profile and then reloaded to get the new one)
3. Load as a module (from a Lua script)